gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Mon Oct 09 16:59:22 EDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.2-all.zip
```

Comment:
I usually pull in only the Gradle binary because of bandwidth restrictions (I'm traveling through the country at the moment, where WiFi isn't always a given). Not sure if running `./gradlew wrapper` would touch any files when changing from `bin` to `all`, but could you check anyway & commit any changes in the JAR, too?

Reply:
Actually, when I ran `/.gradlew assemble`, it changed it to **gradle-4.2-all**.  I believe that is the default configuration type in Android Studio 3.0.

Reply:
It tends to be eager in pulling the most complete distro, yeah… thanks for clarifying!